### DISCUSSION TITLE
I can't log in to dashboard.

### DISCUSSION THREAD

**Discussion by Moonfish90** [ClickHouse/ClickHouse#72122] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ccbc5f08-33fe-47b6-abd5-14b88540240f)

I get the message "LZstring is not defined" after entering the username and password.

**Comment by Trifonov-Artem** [comment]
in google chrome same err

in microsoft edge 
TypeError: Failed to execute 'fetch' on 'Window': Failed to parse URL from http://localhost:8123 ?

**Reply by den-crane** [reply]
LZstring is a js library. You may block it using antivirus / adblock.

https://github.com/ClickHouse/ClickHouse/blob/master/programs/server/dashboard.html#L8C47-L8C56

Check that  http://localhost:8123//js/lz-string.js is available and check developer's tools in your browser.

`curl http://localhost:8123/js/lz-string.js`

**Comment by alexey-milovidov** [comment, marked as answer]
The root cause is found and will be fixed in https://github.com/ClickHouse/ClickHouse/pull/81895

**Comment by Moonfish90** [comment]
It's a browser! Everything opens from Mozilla Firefox and works fine. Thanks for the help.

**Comment by p1xel007** [comment]
@alexey-milovidov 
Is this still not fixed? Changing browser doesn't help for me. 

https://github.com/ClickHouse/ClickHouse/issues/82354

**Reply by alexey-milovidov** [reply]
Fixed in the master build, and will be in 25.6 as well.
